.github/workflows/acceptance-test.yml

```
@@ -24,7 +24,7 @@ jobs:
         # version, because "goenv" can react to it automatically.
         run: |
           echo "Building with Go $(cat .go-version)"
-          echo "::set-output name=go-version::$(cat .go-version)"
+          echo "name=go-version::$(cat .go-version)" >>"$GITHUB_OUTPUT"
```

It looks like you are mixing the syntax style here. 
```suggestion
          echo "go-version=$(cat .go-version)" >>"$GITHUB_OUTPUT"
```

---

Very true, good catch!